docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -696,3 +696,63 @@ The JSON object is of the form:
     "Value" : string
 }
 ```
+
+# 5.0 DICOM Data
+
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
+portion of the REST API namespace, but is defined in a separate section of this 
+document for clarity purposes.
+
+## 5.1 Candidate DICOMs
+```
+GET /candidates/$CandID/$Visit/dicoms
+```
+
+A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
+all the raw DICOM data which have been acquired for that visit. It will return an 
+object of the form:
```

Comment:
would it be a good idea to include an upload date and an optional 'since' parameter to filter it, similarly to what @xlecours did for the images endpoint?

Reply:
yes, good point. But should it be done in this pull request or in a separate one?

Reply:
> would it be a good idea to include an upload date and an optional 'since' parameter to filter it, similarly to what @xlecours did for the images endpoint?

good point. I planned to do it in a separate pull request though. Did you want it here?

Reply:
Sorry for the delayed response.. no, that's fine if you're planning on doing it separately.